tools/snpEff.cwl

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+
+hints:
+  DockerRequirement:
+    dockerPull: quay.io/snpeff:4.3
+
+requirements:
+  - class: InlineJavascriptRequirement
+
+inputs:
+
+  data_dir:
+    type: Directory
+    inputBinding:
+      prefix: "-dataDir"
+      position: 1
```

Comment:
Does snpEff accept the prefixed options in any order? Most tools do, so if that is true then just specify the `prefix` to `inputBinding` and drop the unneeded `position`.